data/kirtankp.json

```diff
@@ -26,6 +32,7 @@
       "icon": "FaGithub"
     },
     {
+      "group": "Primary Socials",
       "group": "Socials Primary",
```

Comment:
Please only have one group per link object. Please remove one so that we can merge your Pull Request.

Reply:
I had resolved it.